identity-service/src/main/java/com/softserve/identityservice/controller/UserController.java

```diff
@@ -25,11 +28,21 @@
     @GetMapping("/activate/{token}")
     public ResponseEntity<Void> verifyAccount(@PathVariable UUID token) throws ServletException {
         userService.activateAccount(token);
-        return new ResponseEntity(HttpStatus.NO_CONTENT);
+        return ResponseEntity.status(HttpStatus.NO_CONTENT).build();
     }
 
     @PutMapping("/admin/block/{id}")
```

Comment:
looks like your trying ot block an admin, why shouldn't it be /users/block/{id} or something like this?

Reply:
because this endpoint must be available only for admin. Check Security Configuration

Reply:
Nit: you also can use ResponseEntity.noContent().build();